src/install/config/python.yaml

```diff
@@ -0,0 +1,750 @@
+  - /docs/agents/python-agent/web-frameworks-servers/uvicorn-web-server
+  - /docs/apm/agents/python-agent/web-frameworks-servers/uvicorn
+
+  - /docs/agents/python-agent/web-frameworks-servers/python-agent-uwsgi-web-server
+  - /docs/python/python-agent-and-uwsgi
+  - /docs/agents/python-agent/hosting-mechanisms/python-agent-and-uwsgi
+  - /docs/agents/python-agent/hosting-mechanisms/python-agent-uwsgi
+  - /docs/apm/agents/python-agent/web-frameworks-servers
+  - /docs/agents/python-agent/web-frameworks-and-servers
+  - /docs/apm/agents/python-agent/web-frameworks-servers/python-agent-uwsgi-web-server
+
+  - /docs/agents/python-agent/web-frameworks-servers/python-agent-waitress-web-server
+  - /docs/python/python-agent-and-waitress
+  - /docs/agents/python-agent/hosting-mechanisms/python-agent-and-waitress
+  - /docs/agents/python-agent/hosting-mechanisms/python-agent-waitress
+  - /docs/apm/agents/python-agent/web-frameworks-servers/python-agent-waitress-web-server
+
+  - /docs/agents/python-agent/web-frameworks-servers/python-agent-web2py-web-framework
+  - /docs/python/python-agent-and-web2py
+  - /docs/agents/python-agent/web-frameworks/python-agent-and-web2py
+  - /docs/agents/python-agent/web-frameworks/python-agent-web2py
+  - /docs/apm/agents/python-agent/web-frameworks-servers/python-agent-web2py-web-framework
+
+
+# -----------------------------------------------------------------
+# Define the categories for each type of question you'll be asking:
+# -----------------------------------------------------------------
+appInfo:
+
+
+
+  - optionType: python-non-web
+    label: "We'll show you how to instrument your non-web Python app"
+    placeholder: 'Make a selection:'
+    options:
+      - value: 'non-web-yes'
+        displayName: 'Yes'
+      - value: 'non-web-no'
+        displayName: 'No'
+
+  - optionType: python-docker
+    label: "We'll give you some basic tips for docker-run"
+    placeholder: 'Make a selection:'
+    options:
+      - value: 'docker-yes'
+        displayName: 'Yes'
+      - value: 'docker-no'
+        displayName: 'No'
+
+  - optionType: python-framework
+    label: "We'll give you information about your framework with New Relic"
+    placeholder: 'Choose one:'
+    options:
+      - value: 'cherrypy'
+        displayName: 'CherryPy'
+      - value: 'fastapi'
+        displayName: 'FastAPI'
+      - value: 'fastcgi'
+        displayName: 'FastCGI'
+      - value: 'paste'
+        displayName: 'Paste'
+      - value: 'scgi'
+        displayName: 'SCGI'
+      - value: 'starlette'
+        displayName: 'Starlette'
+      - value: 'web2py'
+        displayName: 'Web2Py'
+      - value: 'unsupported-asgi-framework'
+        displayName: 'Unsupported ASGI framework'
+      - value: 'unsupported-wsgi-framework'
+        displayName: 'Unsupported WSGI framework'
+
+  - optionType: python-server
+    label: "We'll give you information about your server with New Relic"
```

Comment:
At the very least, I think this label sentence needs to be more active. But I think that it can probably just be deleted.

Reply:
I have the same general feedback for the rest of the label text, but I'm going to skip flagging each one.

Reply:
@paperclypse I can add a few more words like this: "We'll give you information about your set up your server with the Python agent."